crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -43,154 +42,466 @@ mod validate_op_tests;
-                                Outcome::Accepted => {
-                                    summary.accepted += 1;
-                                    put_validation_limbo(
-                                        txn,
-                                        &op_hash,
-                                        ValidationLimboStatus::SysValidated,
-                                    )?;
-                                }
-                                Outcome::AwaitingOpDep(missing_dep) => {
-                                    summary.awaiting += 1;
-                                    // TODO: Try and get this dependency to add to limbo
-                                    //
-                                    // I actually can't see how we can do this because there's no
-                                    // way to get an DhtOpHash without either having the op or the full
-                                    // action. We have neither that's why where here.
-                                    //
-                                    // We need to be holding the dependency because
-                                    // we were meant to get a StoreRecord or StoreEntry or
-                                    // RegisterAgentActivity or RegisterAddLink.
-                                    let status =
-                                        ValidationLimboStatus::AwaitingSysDeps(missing_dep);
-                                    put_validation_limbo(txn, &op_hash, status)?;
-                                }
-                                Outcome::MissingDhtDep => {
-                                    summary.missing += 1;
-                                    // TODO: Not sure what missing dht dep is. Check if we need this.
-                                    put_validation_limbo(
-                                        txn,
-                                        &op_hash,
-                                        ValidationLimboStatus::Pending,
-                                    )?;
-                                }
-                                Outcome::Rejected => {
-                                    summary.rejected += 1;
-                                    if let Dependency::Null = dependency {
-                                        put_integrated(txn, &op_hash, ValidationStatus::Rejected)?;
-                                    } else {
-                                        put_integration_limbo(
-                                            txn,
-                                            &op_hash,
-                                            ValidationStatus::Rejected,
-                                        )?;
-                                    }
-                                }
-                            }
+                let fetched = cascade.retrieve_action(h.clone(), Default::default()).await;
+                tracing::info!(hash = ?h, fetched = ?fetched, "Fetched action for validation");
```

Comment:
All other logs around here are debug, so I'd make this a debug log too.
```suggestion
                tracing::debug!(hash = ?h, fetched = ?fetched, "Fetched action for validation");
```